lib/pbench/agent/tool_meister.py

```diff
@@ -1901,10 +1901,9 @@ def daemon(
         working_dir = tmp_dir
     else:
         working_dir = Path(".")
-    param_key = parsed.key
-    pidfile_name = working_dir / f"{param_key}.pid"
-    d_out = working_dir / f"{param_key}.out"
-    d_err = working_dir / f"{param_key}.err"
+    pidfile_name = working_dir / "tm.pid"
```

Comment:
I think that this means that a user cannot have two tool meisters running on the same system using the same (e.g., the default) configuration...is that a problem?  (Should we be trying to detect collisions rather than just overwriting the PID file??)

Reply:
We always ensure the working directory used by the "create" orchestration path is unique.

Reply:
OK, so it's based on `benchmark_run_dir`, and anyone who runs two workloads out of the same directory at the same time is already in trouble.

Nevertheless, the PID file should not exist before we create it (right??), so it might be worth adding that check.

Reply:
Thankfully that is handled by the `pidfile` module.